packages/slice-machine/src/hooks/useServerState.ts

```diff
@@ -10,15 +10,19 @@ const fetcher = (url: string): Promise<ServerState> =>
 
 const useServerState = () => {
   const { refreshState } = useSliceMachineActions();
+  const handleRefreshState = useCallback(refreshState, []);
```

Comment:
Was this causing an issue? Would it make sense to look for a more global fix, as `useSliceMachineActions` is used in a lot of places

Reply:
linting waring from `eslint-plugin-react-hooks`

Reply:
mainlly the `react-hooks/exhaustive-deps` rule where the function either has to be declared in `useEffect` or passed to the dependency array

Reply:
I was trying things to see what worked, 
i think it was this one https://www.bam.tech/article/how-to-avoid-bugs-in-useeffect
I guess it could be added directly